packages/components/src/__alpha__/SingleSelect/subcomponents/Popover/Popover.tsx

```diff
@@ -28,28 +38,27 @@ export const Popover = ({
     }
   }, [isOpen, popoverRef])
 
-  const { overlayProps } = useOverlayPosition({
-    targetRef: buttonRef,
-    overlayRef: popoverRef,
-    placement: 'bottom start',
-    offset: 0,
-    isOpen: isOpen,
-    shouldFlip: true,
-  })
   return (
     <RACPopover trigger="manual" isOpen={isOpen} onOpenChange={setOpen} ref={racPopoverRef}>
       <div
         // @ts-expect-error - popover attribute is not included in current ts version, ignore type error
         popover="manual"
         ref={popoverRef}
+        // TODO: expect some of these styles to change once we have designs
+        className={styles.popover}
         style={{
-          ...overlayProps,
-          zIndex: 'none',
+          position: 'fixed',
+          top,
+          bottom,
+          insetInlineStart,
+          maxHeight,
+          overflowY: 'auto',
+          left: 'auto',
+          right: 'auto',
           margin: '0',
```

Comment:
You could move the hardcoded styles to the stylesheet to reduce the non dynamic ones here